.isort.cfg

```diff
@@ -0,0 +1,5 @@
+# isort configuration file
+# https://github.com/timothycrosley/isort
+[settings]
+line_length=120
```

Comment:
I'd rather stick w/ 79 as per https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Sorry, we had been using [100 characters for pep8speaks](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/.pep8speaks.yml#L5-L6).
I'll match what we're using there.